test/autoProcess/autoProcess.test.ts

```diff
@@ -106,20 +106,14 @@ describe('options', () => {
     expect(preprocessed.toString?.()).toContain('div{}');
   });
 
-  it('should NOT preprocess preserved languages', async () => {
-    const input = `<div></div><script lang="ld+json">{"json":true}</script>`;
-    const opts = sveltePreprocess({
-      preserve: ['ld+json'],
-      aliases: [['ld+json', 'structuredData']],
-      structuredData() {
-        return { code: '', map: '' };
-      },
-    });
+  it('should NOT preprocess unrecognized languages', async () => {
```

Comment:
It seems that I forgot how the auto preprocessor is working 😅  I didn't remember it's skipping unknown languages 🤔

Reply:
At some point in the past it wasn't, but it's nice that it is now. Saved me from having to implement it!

I think it's not really that it's skipping unknown languages but that it's looking at the `lang` field, so you can specify whatever you want in `type` since it's not using that field. Then since this doesn't specify a `lang` it gets skipped.